packages/react/src/primitives/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { classNameModifierByFlag } from '../shared/utils';
+import {
+  BreadcrumbLinkProps,
+  ForwardRefPrimitive,
+  Primitive,
+  BaseBreadcrumbLinkProps,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { Link } from '../Link';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbLinkPrimitive: Primitive<BreadcrumbLinkProps, 'span'> = (
+  { className, children, href, ...rest },
+  ref
+) => {
+  const { isCurrent, isDisabled } = React.useContext(BreadcrumbsContext);
```

Comment:
These come from the context so customers only need to put `isCurrent` on the Breadcrumbs.Item and not both that and Breadcrumbs.Link